receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml

```diff
@@ -129,6 +129,13 @@ metrics:
       aggregation: cumulative
       monotonic: false
 
+  process.memory.utilization:
+    enabled: false
+    description: Percentage of total physical memory that is used by the process.
+    unit: 1
```

Comment:
@astencel-sumo it's unclear from the docs and unit if the metric is emitted in a fraction of 1 of percentage. I believe it's the same issue for other "utilization" metrics in hostreceiver. Maybe even not well defined in the spec. Can you please help with making it consistent and clear?